test/conformance/service_test.go

```diff
@@ -127,8 +127,8 @@ func TestRunLatestService(t *testing.T) {
 	logger := test.GetContextLogger("TestRunLatestService")
 
 	var imagePaths []string
-	imagePaths = append(imagePaths, strings.Join([]string{test.Flags.DockerRepo, image1}, "/"))
-	imagePaths = append(imagePaths, strings.Join([]string{test.Flags.DockerRepo, image2}, "/"))
+	imagePaths = append(imagePaths, fmt.Sprintf("%s/%s:%s", test.Flags.DockerRepo, image1, test.Flags.Tag))
+	imagePaths = append(imagePaths, fmt.Sprintf("%s/%s:%s", test.Flags.DockerRepo, image2, test.Flags.Tag))
```

Comment:
This is used pretty heavily throughout the tests, maybe a `func ImagePath(repository, image, tag string) string` is warranted? `test/util.go` seems like a good place.

Reply:
Good call. I'll add a commit to this PR and force-push.